firebase-config/firebase-config.gradle

```diff
@@ -78,4 +78,6 @@ dependencies {
     androidTestImplementation 'com.linkedin.dexmaker:dexmaker-mockito:2.25.0'
     androidTestImplementation 'junit:junit:4.12'
     androidTestImplementation "org.skyscreamer:jsonassert:1.5.0"
+
+    api 'joda-time:joda-time:2.10.14'
```

Comment:
This is a pretty heavy dependency, especially given that you don't seem to do any arithmetic on it but just rely on its toString() method, I suggest to use the long directly and use SimpleDateFormat to convert it to an iso string.

Reply:
Removed the dependency and replaced it with SimpleDataFormat. Thanks, I definitely struggled a bit with this and was looking for inputs!